MBINCompiler.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2036
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31205.134
```

Comment:
guess I should probably update my VS to something higher than 15...

Reply:
Yeah, I saw that, but don't think it will break anything.  VS proj and solution files have been backward and forwrd compatible for several versions now.